study.md

```diff
@@ -25,7 +25,8 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+It is a single page web app that eliminates page refreshes by relying on client
+side processing for the entire UI.
```

Comment:
Right, typically making one request on page load and then letting the front end handle routing and view states.